config/exchanges.json

```diff
@@ -675,7 +675,8 @@
       54.87826
     ],
     "parsers": {
-      "exchange": "ENTSOE.fetch_exchange"
+      "exchange": "ENTSOE.fetch_exchange",
```

Comment:
You can't have twice the same key: just remove the ENTSOE one.

Reply:
Ok I thought we agreed to leave ENTSOE in for when it starts working again.

Reply:
Sorry about the misunderstanding. For now the system can't handle multiple parsers. We could do it though.
Maybe we could have an array of parsers here. I'll see if it makes sense.
Let's continue with only your parser for now.

Reply:
Gotcha will do.